src/e2e-test/java/io/cdap/plugin/servicenow/locators/ServiceNowPropertiesPage.java

```diff
@@ -16,6 +16,8 @@
 
 package io.cdap.plugin.servicenow.locators;
 
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
```

Comment:
Why do we need this change ?

Reply:
Removed, they were unused imports. (51534f1cfe832a8f0970c86938db9719f87aa5ab)